LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012-2014 Bithavoc.io - http://bithavoc.io
+Copyright (c) 2012-2019 Bithavoc.io - http://bithavoc.io
```

Comment:
Let’s do 2012. Thanks for fixing this non -sense for us 🤯

Reply:
@bithavoc Good to hear the acceptance of recommendation. Done :)